ias-verify/src/netscape_comment.rs

```diff
@@ -0,0 +1,53 @@
+use crate::utils::{length_from_raw_data, safe_indexing};
+use crate::CertDer;
+use frame_support::ensure;
+use std::convert::TryFrom;
+
+pub struct NetscapeComment<'a> {
+    pub attestation_raw: &'a [u8],
+    pub sig: Vec<u8>,
+    pub sig_cert: Vec<u8>,
+}
+
+pub const NS_CMT_OID: &[u8; 11] = &[
+    0x06, 0x09, 0x60, 0x86, 0x48, 0x01, 0x86, 0xF8, 0x42, 0x01, 0x0D,
+];
+
+impl<'a> TryFrom<CertDer<'a>> for NetscapeComment<'a> {
+    type Error = &'static str;
+
+    fn try_from(value: CertDer<'a>) -> Result<Self, Self::Error> {
```

Comment:
Apart from the fix mentioned below, this code is copied from the `verify_ias_report` function

Reply:
This seems to be more or less duplicated in the worker. How much work would it be to extract this code and share it across the pallet and the worker? Do we already have an existing library that can act as such a vessel?

Reply:
Yes, this is almost an exact duplicate. We have not discussed yet, how we want to share code. We should talk about that in our next call.

Reply:
do we have an issue for this? otherwise we need to create one..

Reply:
Done: https://github.com/integritee-network/worker/issues/323